fossology/fossology
# feat(copyright): add new agent IPRA to FOSSology

## Description

New AGENT `ipra` to detect patent relevant keywords. No keywords were added to this agent it is up-to users to add keywords to Conf page `(ipra.conf)`.

## How to test

* Add few keywords to ipra.conf 
* Install FOSSology and upload a file with same keywords.
* Tool should detect those keywords

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2369"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
